lib/webapp.js

```diff
@@ -335,8 +335,9 @@ const staticHandlers = {
         .then(() => {
           res.status(200).send('plugin added');
         }, (err) => {
-          res.status(400).send('failed to add the plugin: ' + err.message);
-          console.warn(err);
+          res.status(400).send('ZWED0059W - failed to add the plugin: ' + err.message);
+          contentLogger.warn("ZWED0059W", err);
+          console.warn("ZWED0059W", err);
```

Comment:
No need to console.warn here, it is a duplicate

Reply:
Addressed.